helm-chart/renku-gateway/templates/configmap.yaml

```diff
@@ -95,6 +95,20 @@ data:
           ]
           Rule = "PathPrefix(`{{ .Values.gatewayServicePrefix | default "/api/" }}direct/`)"
           Service = "gitlab"
+        
+        [http.routers.gitlabGraphql]
+          entryPoints = ["http"]
+          Middlewares = [
+            "auth-gitlab","general-ratelimit","api"
+            {{- if .Values.global.gitlab.urlPrefix -}}
```

Comment:
Hmm, I think we could just make "/" the default URL prefix and define a middleware that works for all url prefixes, no? I know we've used this exact pattern before and you didn't "invent" it for this PR, but it's not exactly pretty...

Reply:
As discussed offline, it would definitely be nice to harmonize how we handle the GitLab base URL.
Let's move that to a separate issue #493 -- I'll tackle it soon